src/index.js

```diff
@@ -1,3 +1 @@
-import * as UploadAPI from './upload'
-
-export {UploadAPI}
+export * from './upload'
```

Comment:
Cool! But there is one thing: in addition to upload api, there will be rest api, and I would like to somehow divide them into namespaces and do not mess up with the tree shaking as I did :)

I think we can do something like this:
```javascript
import * as UploadAPI from '@uploadcare-api/upload'
import * as RestAPI from '@uploadcare-api/rest'

import {base, fromUrl} from '@uploadcare-api/upload'
```